frontend/src/pages/storage/components/PCStorageContent/PCStorageContent.tsx

```diff
@@ -0,0 +1,65 @@
+import { useQuery } from '@tanstack/react-query';
+import { useEffect, useState } from 'react';
+import ArticleList from '../ArticleList/ArticleList';
+import SearchAndSort from '../SearchAndSort/SearchAndSort';
+import { GetArticlesParams } from '@/apis/articles';
+import { queries } from '@/apis/queries';
+import Pagination from '@/components/Pagination/Pagination';
+import EmptyLetterCard from '@/pages/today/components/EmptyLetterCard/EmptyLetterCard';
+
+interface PCStorageContentProps {
+  baseQueryParams: GetArticlesParams;
+  searchInput: string;
+  onSearchChange: (e: React.ChangeEvent<HTMLInputElement>) => void;
+  sortFilter: 'DESC' | 'ASC';
+  onSortChange: (value: 'DESC' | 'ASC') => void;
+}
+
+export default function PCStorageContent({
+  baseQueryParams,
+  searchInput,
+  onSearchChange,
+  sortFilter,
+  onSortChange,
+}: PCStorageContentProps) {
+  const [currentPage, setCurrentPage] = useState(1);
+
+  const { data: articles, isLoading } = useQuery({
+    ...queries.articles({
+      ...baseQueryParams,
+      page: currentPage - 1,
+    }),
+  });
```

Comment:
p4) 객체로 감싸지 않으면 `...`을 사용할 필요 없을 것 같아요~!

```suggestion
  const { data: articles, isLoading } = useQuery(
    queries.articles({
      ...baseQueryParams,
      page: currentPage - 1,
    }),
  );
```

Reply:
66adb12e4c03c92fd73bbbe1738a49921efe894e